config/routes.rb

```diff
@@ -28,6 +32,10 @@
     end
     resources :comments, only: [:index]
     resources :conferences do
+      get 'conference_domains/edit' => 'conference_domains#edit'
+      get 'conference_domain' => 'conference_domains#show'
+      patch 'conference_domain' => 'conference_domains#update'
```

Comment:
Why don't you use `resource` for this?
`resource :conference_domain`
Since a conference can only have a single custom domain, it should be a singular resource.
See: http://guides.rubyonrails.org/routing.html#singular-resources

Also, the URLs generated are pretty ugly, because they repeat "conference": `conferences/osemdemo/conference_domain`
We could make it nicer, like: `conferences/osemdemo/domain`
With something like: `resource :domain, controller: 'conference_domains'`